intellij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -173,6 +173,39 @@
         level="WARNING"
     />
 
+    <!-- "GraphQL config file present" inspection -->
+    <!--suppress PluginXmlCapitalization, PluginXmlExtensionRegistration -->
+    <localInspection
+        implementationClass="com.apollographql.ijplugin.inspection.ApolloGraphQLConfigFilePresentInspection"
+        groupPathKey="inspection.group.graphql"
+        groupKey="inspection.group.graphql.apolloKotlin"
+        key="inspection.graphQLConfigFilePresent.displayName"
+        enabledByDefault="true"
+        level="ERROR"
```

Comment:
If it is not needed but doesn't harm it should be a warning. 

Suggestion:

```
Apollo Kotlin 4 uses the Gradle tooling API to retrieve the GraphQL configuration and doesn't use the GraphQL config file.
```

If it's an actual error, ERROR is fine of course but sounds like there is a disconnect between the message and the level here.